openedx/openedx-learning
# feat: implement tagging rest api tag object [FC-0030]

## Description

This PR implements the tagging REST API for updating object tags.

## Supporting Information

* Closes https://github.com/openedx/modular-learning/issues/82

## Testing instructions
* Ensure that the tests cover the expected behavior of the view as described in the related issue.
____
Private-ref: [FAL-3474](https://tasks.opencraft.com/browse/FAL-3474)
